MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include youclidbackend/data/ *
```

Comment:
`MANIFEST.in` is used for sdist (source distribution). This will include the necessary data files